common/src/main/java/com/wynntils/core/features/overlays/OverlayPosition.java

```diff
@@ -86,35 +86,35 @@ public String toString() {
 
     public static OverlayPosition getBestPositionFor(
             Overlay overlay, float oldRenderX, float oldRenderY, float offsetX, float offsetY) {
-        Vec2 topLeftOfOverlay = new Vec2(overlay.getRenderX(), overlay.getRenderY());
+        Vec2 referencePoint = new Vec2(
+                overlay.getRenderX() + overlay.getWidth() / 2f, overlay.getRenderY() + overlay.getHeight() / 2f);
 
         // 1. Get the best section (section with the top left point of overlay)
```

Comment:
it seems like you changed this code to use the overlay's center instead of top left (a change i think is good), but didn't update the comment? unless i am confused

Reply:
Yes, sorry